workbench-core/accounts/src/services/accountService.ts

```diff
@@ -175,4 +177,58 @@ export default class AccountService {
 
     return accountMetadata.id;
   }
+
+  private async _getAccountWithoutMetadata(accountId: string): Promise<Account> {
+    const accountEntry = (await this._aws.helpers.ddb
+      .get({ pk: `${resourceTypeToKey.account}#${accountId}` })
+      .execute()) as GetItemCommandOutput;
+
+    if (!accountEntry.Item) {
```

Comment:
does the `ddb.get` always returns a value? Do we need a validation for `accountEntry` like `accountEntry?.item`

Reply:
I believe accountEntry is always valid, but `Item` can be `any | undefined`. I'll double check though.

Reply:
The type checker confirmed that `accountEntry` should always be present.